Frontend/CO.CDP.OrganisationApp/Pages/Organisation/OrganisationOverview.cshtml

```diff
@@ -12,6 +12,23 @@
 <main class="govuk-main-wrapper">
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
+
+            @if (organisationDetails.IsBuyer() && organisationDetails.IsApproved() == false)
+            {
+                <div class="govuk-notification-banner" role="region" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
+                    <div class="govuk-notification-banner__header">
+                    <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+                        Important
+                    </h2>
+                </div>
+                <div class="govuk-notification-banner__content">
+                    <p class="govuk-notification-banner__heading">
+                        Buyer registration pending approval.
+                    </p>
+                </div>
+            </div>
```

Comment:
This section needs to be outdented by one - just so it's clear which <div> tags match with which